src/workloads/execution/TimeSeriesRangeDelete.yml

```diff
@@ -0,0 +1,123 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/server-execution"
+Description: |
+  This test establishes a baseline for time-based range deletes on time-series collections.
+
+  We insert 1000 independent series with 100 buckets in each series, and each bucket has 100
+  documents.
+
+  The buckets have meta fields with values from 0 to 999, each of which has 100 measurements with
+  timestamps within an hour with 36-second intervals. For example, in a bucket, the measurements
+  look like:
+  {t: 2023-01-01T5:00:00, m: 42, ...},
+  {t: 2023-01-01T5:00:36, m: 42, ...},
+  ...
+  {t: 2023-01-01T5:59:24, m: 42, ...}
+
+  Then we delete data that spans across three buckets for each series, causing two partial-bucket
+  deletions and one full-bucket deletion. This tests the use case of data correction where the
+  application deletes data within some time ranges.
+
+GlobalDefaults:
+  dbname: &db test
+  coll: &coll Collection0
+  batchSize: &batchSize 30000
+  nop: &Nop {Nop: true}
+
+Actors:
+- Name: CreateTimeSeriesCollection
+  Type: RunCommand
+  Threads: 1
+  Phases:
+  - Repeat: 1
+    Database: *db
+    Operation:
+      OperationMetricsName: CreateTimeSeriesCollection
+      OperationName: RunCommand
+      OperationCommand:
+        {create: *coll, timeseries: {timeField: "t", metaField: "sensorId"}}
+  - *Nop
+  - *Nop
+  - *Nop
+  - Repeat: 1
+    Database: test
+    Operations:
+    - OperationName: RunCommand
+      OperationCommand:
+        drop: *coll
+
+- Name: InsertData
+  Type: Loader
+  Threads: 1
+  Phases:
+  - *Nop
+  - Repeat: 1
+    Database: *db
+    Collection: *coll
+    Threads: 1
+    CollectionCount: 1
+    DocumentCount: 10000000
```

Comment:
I was trying to use `*` but it didn't work. Glad we can use underscore instead!

We have 1000 devices, each has 100 buckets, and each bucket has 100 measurements.

Reply:
Just found that we have `^NumExpr`. Here's an example for it.

https://github.com/mongodb/genny/blob/master/src/workloads/docs/HelloWorld.yml/#L33-#L35

Let's define three variables like `nSensors`, `nBucketsPerSensor`, `nMeasurementsPerBucket` or somethings similar and use them instead of `10_000_000`.